src/test/java/com/google/sps/servlets/MatchDecisionsServletTest.java

```diff
@@ -207,10 +217,14 @@ public void friendedDecisionMutualMatchFound() throws IOException{
     assertThat(matchInfoEntity2).isNotNull();
 
     List<String> actualMatchesList1 = (List<String>) matchInfoEntity1.getProperty(MATCHES_LIST_PROPERTY);
+    List<String> resultingPotentialMatchesList1 = (List<String>) matchInfoEntity1.getProperty(POTENTIAL_MATCHES_PROPERTY);
     List<String> actualMatchesList2 = (List<String>) matchInfoEntity2.getProperty(MATCHES_LIST_PROPERTY);
+    List<String> resultingPotentialMatchesList2 = (List<String>) matchInfoEntity2.getProperty(POTENTIAL_MATCHES_PROPERTY);
 
     assertThat(actualMatchesList1).containsExactly(TEST_USER_2_ID);
+    assertThat(resultingPotentialMatchesList1).isEqualTo(null);  
     assertThat(actualMatchesList2).containsExactly(TEST_USER_1_ID);
+    assertThat(resultingPotentialMatchesList2).isEqualTo(null);  
```

Comment:
nit: I think there is a `isNull` (or something like that) that you could possibly use.

Reply:
got it!